app/controllers/admin/projects_controller.rb

```diff
@@ -44,7 +47,15 @@ def update
   end
 
   def move_project_to_channel
-    MoveProjectToChannel.new(params[:project_id],params[:channel_id]).call
+    project = Project.find_by(id: params[:project_id])
```

Comment:
`find` raises an error when the record is not found:
`ActiveRecord::RecordNotFound: Couldn't find Project with 'id'=`